.travis.yml

```diff
@@ -7,6 +7,15 @@ os: osx
 # on Traivs.
 env:
   - TEST_DIR=com.google.code.gson-gson
+  - TEST_DIR=org.joda-joda-convert
+  - TEST_DIR=joda-time-joda-time
+
+matrix:
+  allow_failures:
+    # Blocked on https://github.com/JodaOrg/joda-convert/issues/7
```

Comment:
We don't know that.  The next version number is something they'll come up with (maybe 1.7.1, maybe 2.0, etc.), and even that we have no guarantee they will actually release.  The link is sufficient - no reason to add possibly wrong info.